MovieCave - MVVM/MovieCave/MovieCave/Custom UI/CastAndTrailerDetailsView/CastReusable/PopulatedCastView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PopulatedCastView.swift
+//  MovieCave
+//
+//  Created by Admin on 13.01.24.
+//
+
+import UIKit
+
+class PopulatedCastView: UIStackView {
+    
+    //MARK: - Properties
+    private var castViews = [TrailerView]()
```

Comment:
For consistency use the initializer you've used mostly in this project. E.g. private var castViews: [TrailerView] = []